src/sentry/testutils/pytest/json_report_reruns.py

```diff
@@ -1,8 +1,10 @@
-import pytest_jsonreport.plugin
+import pytest_jsonreport.plugin  # type:ignore[import-untyped]
```

Comment:
there's a place to put one-off stubs in `fixtures/stubs-for-mypy`

Reply:
Done. I ran mypy but `tee` means that nothing was red =X
I'm a bit surprised it's not under precommit.

Reply:
it's too slow for pre-commit